modules/drivers/sensors/sen0322/sen0322.js

```diff
@@ -0,0 +1,80 @@
+import Timer from "timer";
+
+const Register = Object.freeze({
+	OXYGEN_DATA: 0x03, ///< register for oxygen data
+	USER_SET: 0x08, ///< register for users to configure key value manually
+	AUTUAL_SET: 0x09, ///< register that automatically configure key value
+	GET_KEY: 0x0A ///< register for obtaining key value
+});
+
+const I2C_ADDR = 0x73;
+
+class SEN0322
+{
+	#io;
+	#key;
+	#onError;
+
+	constructor(options) {
+		this.#io = new options.sensor.io({
+			address: I2C_ADDR,
+			hz: 100_000,
+			...options.sensor
+		});
+		this.#onError = options.onError
+		
+	}
+
+	configure(options)
+	{
+		if (undefined !== options.vol)
+		{
+			const vol = options.vol;
+			const mv = options.mv ?? 0;
+			if (vol < 0 || vol > 25)
+				throw new RangeError("invalid O2 concentration (must be 0-25)");
+			else
```

Comment:
The throw above will always, so the content of this else clause can be directly inline below.